### DISCUSSION TITLE
Define Special Interest Groups

### DISCUSSION THREAD

**Discussion by davidmirror-ops** [flyteorg/flyte#3515] [RFC Incubator]
# Motivation

Flyte community members come from different backgrounds and have specialized skills in various fields. Special Interest Groups are a way to define a "community inside the community" or gather individuals with similar interests to learn and create together. Withing SIGs, a narrower focus can be defined and more specific ideas can be discussed and/or implemented; helping scale the community by leveraging the expertise of its members.

# Proposal
The proposal aims to define the requirements to form a SIG, a template to ease the process of defining a Scope, roles, and overall objectives. Some of the design principles would include: aligning with the new Governance model, making sure there is just enough structure while keeping the process simple, and giving a clear path to anyone looking to join a SIG.

# Implementation
The SIG charter templates can take inspiration from the Kubernetes [Steering Committee docs](https://github.com/kubernetes/community/tree/master/committee-steering/governance), and could be put into test by proposing at least one of the following ideas for SIGs:

* SIG Integrations (extending and improving Flyte's integrations ecosystem)
* SIG Large Models
* SIG Observavility
* What else?

Respond to this discussion with your opinions on the matter

Thanks!

**Comment by pingsutw** [comment]
I like SIG. could we also add some folks in sig to [reviewers or approvers](https://kubernetes.io/docs/contribute/review/for-approvers/)? they can help us review PRs.

**Reply by davidmirror-ops** [reply]
hey @pingsutw that would be the Contributor role as defined in [Governance](https://github.com/flyteorg/community/blob/main/GOVERNANCE.md)

Right now, the process to add Contributors is just a matter of checking requirements and nominating the individual specifying the repo where they would have the privileges mentioned in the doc

**Reply by pingsutw** [reply]
sorry, I didn't know we have governance doc. That's what I'm thinking. thank you for adding that. that's amazing.

**Comment by davidmirror-ops** [comment]
This is completed
